x-pack/plugins/security_solution/public/common/components/exceptions/helpers.test.tsx

```diff
@@ -635,14 +636,18 @@ describe('Exception helpers', () => {
   });
 
   describe('getPrepopulatedItem', () => {
-    test('it returns prepopulated items', () => {
-      const prepopulatedItem = getPrepopulatedItem({
+    const alertDataMock: AlertData = {
```

Comment:
it would likely be better to put this line in a `beforeEach` for the same reasons I mentioned above.